Makefile

```
@@ -0,0 +1,7 @@
+.PHONY: run test fmt
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)